src/server.rs

```diff
@@ -30,6 +80,7 @@ pub async fn start(workers: usize) -> Result<()> {
 
         App::new()
             .wrap(cors)
+            .app_data(airdrop_limiter_1.clone())
```

Comment:
Can this just be `airdrop_limiter.clone()` and then we omit the `airdrop_limiter_1` altogether?

Reply:
no to my knowledge - played around with this for a minute or two and could not satisfy the compiler